genai/text_generation/test_text_generation.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from typing import Generator
+
+from google import genai
+
+import pytest
+
+import textgen_chat_with_txt
+import textgen_chat_with_txt_stream
+import textgen_with_txt
+import textgen_with_txt_img
+import textgen_with_txt_stream
+
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+
+
+@pytest.fixture(autouse=True)
```

Comment:
Truly great work but please simplify this.

Check https://github.com/GoogleCloudPlatform/python-docs-samples/pull/13109